frontend/console/e2e/global-setup.ts

```diff
@@ -8,19 +8,25 @@ const globalSetup = async (config: FullConfig) => {
   const page = await context.newPage()
   await page.goto('http://localhost:8899/modules')
 
-  console.log('Waiting for modules to load...')
+  console.log('Waiting for modules to be ready...')
   const moduleNames = ['time', 'echo', 'cron', 'http', 'pubsub']
   await page.waitForFunction(
     (modules) => {
-      const loadedModules = modules.filter((module) => document.querySelector(`li#module-tree-module-${module}`) !== null)
-      console.log('Loaded modules:', loadedModules.join(', '))
-      return loadedModules.length === modules.length
+      const readyModules = modules.filter((module) => {
+        const moduleElement = document.querySelector(`li#module-tree-module-${module}`)
+        if (!moduleElement) return false
+
+        const greenDot = moduleElement.querySelector('.bg-green-400')
+        return greenDot !== null
+      })
```

Comment:
This will make the "ready check" make sure the green dots are visible for all the modules we're starting.